src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -56,6 +68,10 @@ const rpc::PlacementGroupTableData &GcsPlacementGroup::GetPlacementGroupTableDat
   return placement_group_table_data_;
 }
 
+rpc::Bundle *GcsPlacementGroup::GetMutableBundle(int bundle_index) {
+  return placement_group_table_data_.mutable_bundles(bundle_index);
```

Comment:
This method seems unnecessary.

Reply:
We need to modify bundle by this function.

Reply:
Ah, my bad. I thought it was the method for `gcsPlacementGroupManager`.